src/fontra/views/editor/edit-behavior.js

```diff
@@ -84,6 +84,7 @@ class EditBehavior {
       behavior
     );
     this.pointEditFuncs = pointEditFuncs;
+    this.participatingPointIndices = participatingPointIndices;
```

Comment:
Please don't edit this module until we've together established whether something should change.

Reply:
Sure, any other idea how I get get the participatingPointIndices?

Reply:
You shouldn't need them: everything should be handled by EditBehavior itself. You just pass it the selection. See the arrow key nudge code, it should be very similar to that.

Reply:
I tried to understand the arrow key nudge code, but it's very complex. The arrow key tool has a delta which is equal for every point, but in this case every point has a different delta, depending on the pinPoint and the transformation. It was way too complicated for me to figure out how to find a solution based on the arrow key tool / EditBehavior. I only need the effected point indices. Please don't get me wrong, I do my best to find a solution, but it's sometimes not easy. 
I may not understand the code enough to find the best solution. I am very sorry.

Reply:
Ok, let's work on it together tomorrow.